docs/guides/completions.md

```diff
@@ -85,9 +87,13 @@ stream = Completion.create(
 )
 
 for response in stream:
-    if response.output:
-        print(response.output.text, end="")
-        sys.stdout.flush()
+    try:
+        if response.output:
+            print(response.output.text, end="")
+            sys.stdout.flush()
+    except: # an error occurred
+        print(stream.text) # print the error message out 
```

Comment:
hmm, this doesn't seem right to me, shouldn't we print `response.error` or something? printing from `stream` doesn't feel right